codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/RubySymbolProvider.java

```diff
@@ -279,9 +279,9 @@ public Symbol timestampShape(TimestampShape shape) {
     @Override
     public Symbol listShape(ListShape shape) {
         Symbol member = toSymbol(model.expectShape(shape.getMember().getTarget()));
-        String rbsType = "Array[" + member.getProperty("rbsType").get() + "]";
-        String yardType = "Array<" + member.getProperty("yardType").get() + ">";
-        return createSymbolBuilder(shape, getDefaultShapeName(shape, "List__"), rbsType, yardType, moduleName)
+        String rbsType = "::Array[" + member.getProperty("rbsType").get() + "]";
```

Comment:
Out of scope of this PR - but I think related to my comment from previous PR - I think maybe we shoudl define a RubyType class and create instances for things like Array<Type>. that will know how to do the doc types, rbs, contarints, ect....  I think this applies here and in the middleware and possibly in other places as well.

Reply:
I'm not sure what that would look like... But yeah we can potentially do that outside this change.